notebooks/121-convert-to-openvino/121-convert-to-openvino.ipynb

```diff
@@ -0,0 +1,1648 @@
+      "Find more information about API v2.0 and IR v11 at https://docs.openvino.ai/2023.0/openvino_2_0_transition_guide.html\n",
+      "[ SUCCESS ] Generated IR version 11 model.\n",
+      "[ SUCCESS ] XML file: /home/asuvorov/projects/openvino/openvino_notebooks/notebooks/121-convert-to-openvino/model/resnet.xml\n",
+      "[ SUCCESS ] BIN file: /home/asuvorov/projects/openvino/openvino_notebooks/notebooks/121-convert-to-openvino/model/resnet.bin\n",
+      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
+      "To disable this warning, you can either:\n",
+      "\t- Avoid using `tokenizers` before the fork if possible\n",
+      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
+      "[ INFO ] The model was converted to IR v11, the latest model format that corresponds to the source DL framework input/output format. While IR v11 is backwards compatible with OpenVINO Inference Engine API v1.0, please use API v2.0 (as of 2022.1) to take advantage of the latest improvements in IR v11.\n",
+      "Find more information about API v2.0 and IR v11 at https://docs.openvino.ai/2023.0/openvino_2_0_transition_guide.html\n",
+      "[ SUCCESS ] Generated IR version 11 model.\n",
+      "[ SUCCESS ] XML file: /home/asuvorov/projects/openvino/openvino_notebooks/notebooks/121-convert-to-openvino/model/resnet.xml\n",
+      "[ SUCCESS ] BIN file: /home/asuvorov/projects/openvino/openvino_notebooks/notebooks/121-convert-to-openvino/model/resnet.bin\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Model Optimizer CLI\n",
+    "\n",
+    "! mo --input_model model/resnet.onnx --layout \"nchw->nhwc\" --output_dir model\n",
+    "\n",
+    "# alternatively\n",
+    "! mo --input_model model/resnet.onnx --source_layout nchw --target_layout nhwc --output_dir model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Python conversion API\n",
+    "from openvino.tools.mo import convert_model\n",
+    "\n",
+    "\n",
+    "ov_model = convert_model(ONNX_CV_MODEL_PATH, layout=\"nchw->nhwc\")\n",
+    "\n",
+    "ov_model = convert_model(ONNX_CV_MODEL_PATH, source_layout=\"nchw\", target_layout=\"nhwc\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Specifying Mean and Scale Values\n",
+    "\n",
+    "Model conversion API represented parameters to specify the values: `mean_values`, `scale_values`, `scale`. Using these parameters, model conversion API embeds the corresponding preprocessing block for mean-value normalization of the input data and optimizes this block so that the preprocessing takes negligible time for inference."
```

Comment:
```suggestion
    "Model conversion API has the following parameters to specify the values: `mean_values`, `scale_values`, `scale`. Using these parameters, model conversion API embeds the corresponding preprocessing block for mean-value normalization of the input data and optimizes this block so that the preprocessing takes negligible time for inference."
```